scripts/generate_known_standard_library.py

```diff
@@ -0,0 +1,91 @@
+"""Vendored from [scripts/mkstdlibs.py in PyCQA/isort](https://github.com/PyCQA/isort/blob/e321a670d0fefdea0e04ed9d8d696434cf49bdec/scripts/mkstdlibs.py).
+
+Only the generation of the file has been modified for use in this project.
+"""
+
+from pathlib import Path
+
+from sphinx.ext.intersphinx import fetch_inventory
```

Comment:
Do we need to add `sphinx` as a dep?

Reply:
We would have to do the parsing ourselves.
I could do it if you think it is worth it.

Reply:
Isort does not have sphinx as a dependency.
I can add it anyway.

Reply:
But, did you have to install Sphinx for this to work? I'm confused.

Reply:
It's alright, I can always fix it later if needed.

Reply:
Yes, I guess they install it manually to run every new Python release.

Reply:
I added it.